boot/boot_serial/src/boot_serial.c

```diff
@@ -310,6 +387,130 @@ bs_list(char *buf, int len)
     boot_serial_output();
 }
 
+#ifdef MCUBOOT_SERIAL_IMG_GRP_IMAGE_STATE
+/*
+ * Set image state.
+ */
+static void
+bs_set(char *buf, int len)
+{
+    /*
+     * Expected data format.
+     * {
+     *   "confirm":<true for confirm, false for test>
+     *   "hash":<hash of image (OPTIONAL for single image only)>
+     * }
+     */
+    struct image_header hdr;
+    uint8_t tmpbuf[64];
```

Comment:
This can be moved to the block with FIH_CALL, as it is only used there.

Reply:
Fixed